slyd/app/components/annotations-plugin/component.js

```diff
@@ -584,6 +593,28 @@ export default Ember.Component.extend({
         }
     },
 
+    setIgnores: function() {
```

Comment:
That's correct.

It should be set with:

    this.set('pluginState.ignores', ignores);
    this.updateData('pluginState');

This is to allow it to better handle updates to ember in the future.